test/conformance/gateway_conformance_test.go

```diff
@@ -24,12 +24,25 @@ import (
 var skippedTestsForTraditionalRoutes = []string{
 	// core conformance
 	tests.HTTPRouteHeaderMatching.ShortName,
+	// There is an issue with KIC when processing this scenario.
+	// TODO: https://github.com/Kong/kubernetes-ingress-controller/issues/6136
+	tests.GRPCRouteListenerHostnameMatching.ShortName,
+	// tests.GRPCRouteHeaderMatching.ShortName and tests.GRPCExactMethodMatching.ShortName may
+	// have some conflicts, skipping either one will still pass normally.
```

Comment:
Can we create an issue for this and add a TODO here?

Reply:
Sure.

Reply:
Added. https://github.com/Kong/kubernetes-ingress-controller/issues/6144